### DISCUSSION TITLE
Self nomination for contributor membership

### DISCUSSION THREAD

**Discussion by robdimsdale** [paketo-buildpacks/feedback#91] [Builder Team]
I've submitted a few PRs to builders, and I intend on continuing with ongoing contribution. I'd like to become a member of the contributors team to make it easier to submit more PRs as well as re-run builds, etc.

Thoughts @paketo-buildpacks/builders-maintainers ?

Thanks,
Rob

**Comment by ryanmoran** [comment]
Congrats @robdimsdale on becoming a Builders Contributor!
